cppframework/Foo.cpp

```diff
@@ -2,9 +2,6 @@
 #include <string>
 #include <iostream>
 
-#define EXPORT __attribute__((visibility("default")))
```

Comment:
Also, hiding symbols by default is going to be a problem for people migrating libraries from Xcode to CMake. At very least there should be a way to disable hiding symbols other than editing ios.cmake...